sklearn/utils/fixes.py

```diff
@@ -335,3 +335,20 @@ def __call__(self, *args, **keywords):
             return self.func(*args, **kwargs)
 else:
     from functools import partial
+
+
+if np_version < (1, 6, 2):
+    # Allow bincount to accept empty arrays
+    # https://github.com/numpy/numpy/commit/40f0844846a9d7665616b142407a3d74cb65a040
+    def bincount(x, weights=None, minlength=None):
+        x = np.asarray(x, dtype=np.intp)
```

Comment:
Yes but does my proposal not work? Len should work with all these



On Feb 1, 2015, 17:58, at 17:58, "Loïc Estève" <notifications@github.com> wrote:
>> @@ -335,3 +335,20 @@ def __call__(self, *args, **keywords):
>>              return self.func(*args, **kwargs)
>>  else:
>>      from functools import partial
>> +
>> +
>> +if np_version < (1, 6, 2):
>> +    # Allow bincount to accept empty arrays
>> +    #
>https://github.com/numpy/numpy/commit/40f0844846a9d7665616b142407a3d74cb65a040
>> +    def bincount(x, weights=None, minlength=None):
>> +        x = np.asarray(x, dtype=np.intp)
>
>I did that because np.bincount accept array_like inputs for its first
>argument.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/scikit-learn/scikit-learn/pull/4193/files#r23899016